docs/install.md

```diff
@@ -0,0 +1,150 @@
+`HW`:
+
+```bash
+export SGX_MODE=HW
+```
+
+The remainder of this section provides information about preparing to
+run Private Data Objects using SGX in hardware mode. Specifically, there
+are steps that must be taken to enable attestation of the hardware
+platform using the Intel Attestation Service (IAS).
+
+### Update the BIOS
+
+It may be advisable to update your BIOS to pick up any recent
+patches. For example, if you want to run PDO on an Intel NUC, you can
+visit the
+[Intel Download Center](https://downloadcenter.intel.com/)
+(and select `Mini PCs`) to find the latest BIOS release. Other vendors
+may provide similar updates.
+
+Note: If the Trusted Computing Base (TCB) of the platform performing an
+attestation is outdated, IAS can still verify an attestation; however,
+the verification may include a `GROUP_OUT_OF_DATE` or
+`CONFIGURATION_NEEDED` quote status. It is up to the verifier to decide
+whether to trust an attestation from a platform whose TCB has been
+identified as outdated. Note that PDO will flag this situation with a
+warning.
+
+### Create an IAS Authentication Key
+
+Access to IAS requires that a service provider create an identity (SPID)
+and a client authentication key. To create the SPID and the key you
+first need to register with the
+[Intel Developer Portal](https://api.portal.trustedservices.intel.com/developer).
+
+Once your registration is complete subscribe for a
+[linkable quote](https://api.portal.trustedservices.intel.com/EPID-attestation)
+to create the client authentication key. The key will be available from
+your profile page.
+
+Assuming that existing keys are found in `${PDO_SGX_KEY_ROOT}` (which
+defaults to the directory
+`${PDO_SOURCE_ROOT}/build/keys/sgx_simulation`),
+copy the key from your profile page into a file called
+`${PDO_SGX_KEY_ROOT}/sgx_spid_api_key.txt`.
+
+In addition, your SPID can be found on your developer portal profile
+page. Copy the contents of the SPID into the file
+`${PDO_SGX_KEY_ROOT}/sgx_spid.txt`.
+
+### Install the SGX Kernel Driver (Hardware Support)
+
+SGX can run in either simulation or hardware mode. No kernel driver is
+required to run in simulation mode. However, if you plan to run with SGX
+hardware support, it is necessary to install the SGX kernel driver. The
+following commands will download and install v2.4 of the SGX kernel
+driver:
+
+```bash
+wget https://download.01.org/intel-sgx/linux-2.4/ubuntu18.04-server/sgx_linux_x64_driver_778dd1f.bin
+sudo /bin/bash sgx_linux_x64_driver_778dd1f.bin
+```
+
+**NOTE: if you plan to use Docker to build and run PDO, the kernel
+driver will be installed into your containers automatically. **
+
+## Configuration
+
+PDO uses a number of environment variables to control build,
+installation and operation. While PDO should build and run with only the
+default values, some configuration may be required. A list of commonly
+used environment variables is available [here](environment.md).
```

Comment:
I think we could be explicit if the env should be set up before or after build. Specifically, it is preferable to avoid "some configuration may be required"

Reply:
Agreed but that is a separate task. See #212.